### PR TITLE
Updated all apps to use isolated diskcache

### PR DESCRIPTION
**Pull request by JohnLXiang**

### CODE REVIEW

File: 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java [hunk 1]
```diff
@@ -334,60 +342,81 @@ private void initialize() {
...
-                    System.out.println("ERROR: Cannot initialize Map Fragment" + error.toString());
+        // Set path of isolated disk cache
+        String diskCacheRoot = Environment.getExternalStorageDirectory().getPath()
+                + File.separator + ".isolated-here-maps";
+        // Retrieve intent name from manifest
+        String intentName = "";
+        try {
+            ApplicationInfo ai = getPackageManager().getApplicationInfo(m_activity.getPackageName(), PackageManager.GET_META_DATA);
+            Bundle bundle = ai.metaData;
+            intentName= bundle.getString("INTENT_NAME");
```
Comment (AmbroseLi) [inline 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java]: Fix up spacing

Comment (JohnLXiang) [reply 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java]: Yes, just noticed after pushed them in. Working on the formatting now...

File: 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java [hunk 2]
```diff
@@ -334,60 +339,83 @@ private void initialize() {
...
+                + File.separator + ".isolated-here-maps";
+        // Retrieve intent name from manifest
+        String intentName = "";
+        try {
+            ApplicationInfo ai = getPackageManager().getApplicationInfo(m_activity.getPackageName(),
+                    PackageManager.GET_META_DATA);
+            Bundle bundle = ai.metaData;
+            intentName = bundle.getString("INTENT_NAME");
+        } catch (PackageManager.NameNotFoundException e) {
+            Log.e(this.getClass().toString(), "Failed to find intent name, NameNotFound: " + e.getMessage());
```
Comment (JohnLXiang) [inline 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java]: This line looks fine if you switch to Unified view.

File: 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java [hunk 3]
```diff
@@ -200,6 +204,7 @@ public VenueFloorsController(Context context, VenueMapFragment venueLayer,
 
             if (m_venueMapFragment.getSelectedVenue() != null) {
                 onVenueSelected(m_venueMapFragment.getSelectedVenue());
+
```
Comment (sschuberth) [inline 3d-venues-and-indoor-routing/app/src/main/java/com/here/android/example/venue/Venue3dActivity.java]: Nit: Superfluous addition of blank line.

File: geocoder-and-reverse-geocoder/app/src/main/java/com/here/android/example/geocoding/MainView.java [hunk 4]
```diff
@@ -47,18 +53,42 @@ public MainView(Activity activity) {
...
-         * prerequisite.
-         */
-        MapEngine.getInstance().init(m_activity, new OnEngineInitListener() {
-            @Override
-            public void onEngineInitializationCompleted(Error error) {
-                Toast.makeText(m_activity, "Map Engine initialized with error code:" + error,
-                        Toast.LENGTH_SHORT).show();
-            }
-        });
+
```
Comment (sschuberth) [inline geocoder-and-reverse-geocoder/app/src/main/java/com/here/android/example/geocoding/MainView.java]: Nit: Superfluous addition of blank line.

File: turn-by-turn-navigation/app/src/main/java/com/here/android/example/guidance/MapFragmentView.java [hunk 5]
```diff
@@ -232,17 +260,21 @@ private void startNavigation() {
...
         alertDialogBuilder.setMessage("Choose Mode");
-        alertDialogBuilder.setNegativeButton("Navigation",new DialogInterface.OnClickListener() {
+        alertDialogBuilder.setNegativeButton("Navigation", new DialogInterface.OnClickListener() {
             public void onClick(DialogInterface dialoginterface, int i) {
                 m_navigationManager.startNavigation(m_route);
                 m_map.setTilt(60);
-            };
+            }
+
+            ;
```
Comment (sschuberth) [inline turn-by-turn-navigation/app/src/main/java/com/here/android/example/guidance/MapFragmentView.java]: Nit: `;` should come directly after `}` above.

File: turn-by-turn-navigation/app/src/main/java/com/here/android/example/guidance/MapFragmentView.java [hunk 6]
```diff
@@ -232,17 +260,21 @@ private void startNavigation() {
...
-        alertDialogBuilder.setPositiveButton("Simulation",new DialogInterface.OnClickListener() {
+        alertDialogBuilder.setPositiveButton("Simulation", new DialogInterface.OnClickListener() {
             public void onClick(DialogInterface dialoginterface, int i) {
-                m_navigationManager.simulate(m_route,60);//Simualtion speed is set to 60 m/s
+                m_navigationManager.simulate(m_route, 60);//Simualtion speed is set to 60 m/s
                 m_map.setTilt(60);
-            };
+            }
+
+            ;
```
Comment (sschuberth) [inline turn-by-turn-navigation/app/src/main/java/com/here/android/example/guidance/MapFragmentView.java]: Nit: `;` should come directly after `}` above.

**Comment by JohnLXiang** [comment]
Will ask original author to make change.

**Comment by sschuberth** [comment]
Superseded by https://github.com/heremaps/here-android-sdk-examples/pull/96.

**Pull request closed without merging**